suite/sanity/loss_recover.go

```diff
@@ -0,0 +1,173 @@
+package sanity
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/gravitational/robotest/infra/gravity"
+	"github.com/gravitational/trace"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// K8S API master node
+	nodeApiMaster = "apim"
+	// Gravity Site master node
+	nodeGravitySiteMaster = "gsm"
```

Comment:
You should also use `cluster` instead of `gravity site` - this terminology has been obsoleted and should be slowly factored out